File: runtime/Jlib/nil.java
package bigloo;

public final class nil extends cnst
{
  public static final nil nil= new nil( 0 );

  private nil( final int  n )
  {
    super( n );
  }

  public void write( final output_port  p )
  {
    p.write( "()" );
  }
}

File: runtime/CSlib/nil.cs
namespace bigloo
{
  public sealed class nil: cnst
  {
    public static readonly nil _nil= new nil( 0 );

    private nil( int  n )
      : base( n )
    {
    }

    public override void write( output_port  p )
    {
      p.write( "()" );
    }
  }
}
